example/backend/request.js

```diff
@@ -95,9 +95,6 @@ function validateRequest(request) {
 
 function validateResponse(response) {
   const { headers, status } = response;
-  if (status !== 200) {
-    return false;
-  }
   if (!isJSON(headers.get('content-type'))) {
```

Comment:
I find it odd that we are passing in the content type string to verify that response is of JSON type. We should pass in the entire response and do the JSON verifying in the method body.

Reply:
So, we call [.json()](https://developer.mozilla.org/en-US/docs/Web/API/Body/json) which throws on invalid JSON from `requestWithCORS`. This is just to halt any further attempt to deal with the response if the `content-type` is invalid.

Reply:
I see. Can we rename method. IsJsonContentTypeHeader?